modules/ROOT/pages/platform/security.adoc

```diff
@@ -255,3 +255,79 @@ TLS v1.2:
 * `TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (RFC5289)`
 * `TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256 (RFC7905)`
 * `TLS_DHE_RSA_WITH_AES_256_GCM_SHA384 (RFC5288)`
+
+== Customer managed keys
+
+label:AuraDB-Enterprise[]
+label:AuraDS-Enterprise[]
+label:beta[]
+
+[CAUTION]
+====
+
+This feature has been released as a public GA for AuraDB Enterprise and AuraDS Enterprise for AWS managed keys. 
+GCP's Cloud Key Management is in public beta, and support for Azure's Key Vault is coming soon.
+====
+
+For more control over key operations than the standard Neo4j encryption, use customer-managed keys. 
```

Comment:
If you're going without the hyphen, you should change this too.

Reply:
Thank you! Well spotted